services/webhook/tests/test_harvest_bot_findings.py

```
@@ -0,0 +1,157 @@
+        assert f is not None
+        assert f.line == 61
+        assert f.outdated is True
+
+    def test_severity_scale(self):
+        for label, expect in (
+            ("Critical", "critical"), ("Major", "high"),
+            ("Minor", "medium"), ("Trivial", "low"),
+        ):
+            body = f"_🎯 Functional Correctness_ | _🟠 {label}_\n\n**T.**\n"
+            f = parse_coderabbit_comment("o/r", _cr_raw(body=body))
+            assert f is not None and f.severity == expect, label
```

Ruff (0.15.20)

[warning] Assertion should be broken down into multiple parts

(PT018)